CHANGELOG.md

```diff
@@ -9,6 +9,24 @@ NEW FEATURES:
 
     Unlike the existing `precondition` and `postcondition` blocks, Terraform will not halt execution should the scoped data block fail or error or if any of the assertions fail. 
     This allows practitioners to continually validate the state of their infrastructure outside the usual lifecycle management cycle. 
+    
+* `import` blocks for importing infrastructure: Root module authors can now use the `import` block to declare their intent that Terraform adopt an existing resource. 
+  
+    Import is now a plannable action, and is processed as part of a normal plan. Running `terraform plan` will show a summary of the resources that Terraform has planned to import, along with any other plan changes. These may include updates and replacements to the imported resource, if the configuration for that resource differs from the state of the remote system. 
+
+    The existing `terraform import` CLI command has not been modified. Running the CLI command directly imports a resource's state into the state file, while using the `import` block and running `terraform plan` will not modify the state until `terraform apply` is run.
```

Comment:
I think we should call that all future import related functionality will be based off the `import{}` block,  and we will not be doing any more work on the `terraform import` cli command. I think its also worth calling out that the current `terraform import` CLI command is not under any compatibility promises.

Reply:
This will be in the docs